core/internal/testdata/multiword_v1_runlog.json

```diff
@@ -0,0 +1,28 @@
+{
+  "initiators": [
+    {
+      "type": "runlog"
+    }
+  ],
+  "tasks": [
+    {"type": "httpgetwithunrestrictednetworkaccess", "params": {"get": "https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD"}},
```

Comment:
If it's external it doesnt need to be unrestricted

Reply:
that external one is just a place holder, it gets rewritten with a mock server url

Reply:
OK perhaps we could create something more obvious as a placeholder name then? This looks like it might do an actual request.

Reply:
Sure done